Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml

```diff
@@ -1428,25 +1440,192 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "67":
+    id: "67"
+    taskid: 7212ec41-fb6c-4d2b-86c1-140108b79c04
+    type: regular
+    task:
+      id: 7212ec41-fb6c-4d2b-86c1-140108b79c04
+      version: -1
+      name: Save account usernames
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "48"
+      - "9"
+      - "50"
+      - "16"
+      - "11"
+      - "32"
+      - "42"
+      - "55"
+      - "64"
+    scriptarguments:
+      key:
+        simple: Usernames
+      value:
+        complex:
+          root: inputs.Username
```

Comment:
This is taking the ${inputs.Username} puts it in a new key called ${Usernames}.
We can't change the name of the input, and since I created a distinction between usernames **with** domains and usernames **without** domains, I decided to name it in plural form which is more correct in this case

Reply:
Should we not change the input format from ${Usernames} so it takes it from filters and transformers? its bad practice to use ${} notation isn't it?

Reply:
I didn't explain it well. What I meant is that we are taking the data inside inputs.Username and putting in a new Usernames key. The "simple" is just the name of the new key. The data is still taken in the complex way:
![image](https://github.com/demisto/content/assets/43602124/29508766-e0fd-40f2-a853-4572d38263bd)

And yes we should try to avoid ${} as much as possible :)